Cargo.toml

```diff
@@ -107,6 +103,7 @@ jf-relation = { git = "https://github.com/EspressoSystems/jellyfish", tag = "0.4
 jf-utils = { git = "https://github.com/EspressoSystems/jellyfish", tag = "0.4.5" }
 libp2p = { version = "0.53", default-features = false }
 log-panics = { version = "2.0", features = ["with-backtrace"] }
+snafu = "0.8"
```

Comment:
removing it from workspace was intentional. Only sequencer crate now uses it because of query service